tests/__tests__/components/Record.spec.js

```diff
@@ -110,6 +110,7 @@ describe('Record', function () {
         }
     });
 
+    //TODO (after migrating to React 18): Create test cases for different extension (operator and supplier cases)
```

Comment:
I would put this rather to ticket and not here.

Reply:
If we fix tests we will know right away which tests are using which extensions.

Reply:
Ok thanks, it will be dealt in #54